docs/network-flow-visibility.md

```diff
@@ -623,6 +631,37 @@ a new manifest:
 make manifest
 ```
 
+##### ClickHouse Performance Configuration
+
+The ClickHouse throughput depends on two factors - the storage size of the ClickHouse
+and the time interval between the batch commits to the ClickHouse. Larger storage
+size and longer commit interval provide higher throughput.
+
+Grafana flow collector supports the ClickHouse in-memory deployment with limited
+storage size. This is specified in [clickhouse.yml][clickhouse_manifest_yaml].
+The default value of storage size for the ClickHouse server is 8 GiB. Users
+can expect a linear growth in the ClickHouse throughput when they enlarge the
+storage size. To deploy the ClickHouse with a different storage size, please
+modify the `sizeLimit` in the following section.
+
+```yaml
+- name: clickhouse-storage-volume
+  emptyDir:
+    medium: Memory
+    sizeLimit: 8Gi
+```
+
+After making the change, run the following command to generate a new manifest:
+
+```shell
+./hack/generate-manifest-flow-visibility.sh > build/yamls/flow-visibility.yml
+```
```

Comment:
I would expect most users to download the released manifest and not run `generate-manifest.sh` directly.

Reply:
The paragraph here aims at providing instructions on how user can get a new yaml file after they make some configuration changes.

We already have some instructions for released manifest here: https://github.com/antrea-io/antrea/blob/f93ae373b75add2ccc30f6a14bf26fa921f600d2/docs/network-flow-visibility.md?plain=1#L523-L537